### ISSUE TITLE
Grammar fix

### ISSUE THREAD

**Issue by impressivewebs** [yeoman/yeoman#687]
Since there's probably nothing else I could contribute to this amazing way-over-my-head project, I'll just offer a grammar tip:

The readme says (under "what am I?"):

"Yeoman is a robust and opinionated client-side stack, **comprised of** tools and frameworks that can help developers quickly build beautiful web applications."

This should say:

"Yeoman is a robust and opinionated client-side stack, **comprising** tools and frameworks that can help developers quickly build beautiful web applications."

Basically, you should never say "comprised of". Either say "composed of" or else say "comprising". 

See: http://grammar.quickanddirtytips.com/comprise-versus-compose.aspx

The "why yeoman" page on the website also has this mistake:

http://yeoman.io/whyyeoman.html

Now, I'll take my nitpicky, grammar-nazi opinion and crawl back into my hole.... :)
